lib/bom.js

```diff
@@ -130,13 +131,13 @@ declare class MutationRecord {
 declare class MutationObserver {
     constructor(callback: (arr: Array<MutationRecord>, observer: MutationObserver)=>any): void;
     observe(target: Node, options: {
-        childList: boolean;
-        attributes: boolean;
-        characterData: boolean;
-        subtree: boolean;
-        attributeOldValue: boolean;
-        characterDataOldValue: boolean;
-        attributeFilter: Array<string>;
+        childList?: ?boolean;
```

Comment:
let's make all of these optional but not nullable (`childList?: boolean`). even though the API might not care, i don't think we need to be that loose.